.gitignore

```diff
@@ -0,0 +1,9 @@
+*.iml
+.gradle
+/local.properties
+/.idea/workspace.xml
```

Comment:
Please remove "/" in front since this is not needed. .gitignore understands that if its just a name that its in the root repo directory.